h2o-r/tests/testdir_jira/runit_pubdev_4659.R

```diff
@@ -0,0 +1,42 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+
+test_4659 <- function() {
+    prostate = read.csv(locate("smalldata/logreg/prostate.csv"))
+    prostate$our_factor<-as.factor(paste0("Q",c(rep(c(1:380),1))))
+
+    prostate<-as.h2o(prostate)
+    prostate$weight<-1
```

Comment:
random weights would be better (future-proof)

Reply:
Yes, that is true, but for the test, it is important the presence of the column, it does not matter what it contains.